py3status/modules/taskwarrior.py

```diff
@@ -9,6 +9,7 @@
 
 Format placeholders:
     {task} active tasks
+    {num_tasks} number of active tasks
```

Comment:
Hi! Thanks for this PR. I don't really like `num_` prefix... I want to rename a placeholder first ... so we can support your new placeholder too... Can you look at `class Meta` example in `deadbeef.py` to rename `task` to `descriptions` and use `num_tasks` as `tasks`? Does that sounds good to you?

Reply:
Hi,

Thanks for your suggestions. I think I understood what u mean and renamed the format variables accordingly. Now it should work with *task*, *descriptions* and *tasks*. I also added a class which implements the functionality of a specific format identifier. That way, more format identifiers could be added easily later on (such as, for instance, to display only the description of the task with the highest urgency). Appreciate feedback greatly appreciated!